packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -744,4 +744,40 @@ MeetingUtil.handleDeviceLogging = (devices = []) => {
   });
 };
 
+MeetingUtil.endMeetingForAll = (meeting, options = {}) => {
+  if (meeting.meetingState === FULL_STATE.INACTIVE) {
+    // TODO: clean up if the meeting is already inactive
+    return Promise.reject(new MeetingNotActiveError());
+  }
+
+  const defaultOptions = {
+    locusUrl: meeting.locusUrl
+  };
+
+  const leaveOptions = {...defaultOptions, ...options};
+
+  return meeting.meetingRequest
+    .endMeetingForAll(leaveOptions)
+    .then((response) => Promise.resolve(response))
+    .then(() => {
+      if (options.moveMeeting) {
```

Comment:
does it make sense to have a `moveMeeting` option for "end meeting for all"?

Reply:
Removed moveMetting from here